transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/Tree.java

```diff
@@ -17,7 +18,7 @@ public class Tree {
     // intents organized in order of their turn in their respective transcript
     public Tree(ArrayList<ArrayList<ArrayList<String>>> transcripts) {
         // root of tree
-        this.treeRootNode = new Node("Insufficient Data", null);
+        this.treeRootNode = new Node("Insufficient Data");
         Node start = this.treeRootNode;
```

Comment:
Possibly add a comment specifying where the parameter "transcripts" Is being used